nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -525,6 +525,14 @@ systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/var/www" ];
      automatically if <literal>stateVersion</literal> is 20.09 or higher.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      <literal>udpt</literal> experienced a complete rewrite from C++ to rust. The configuration format changed from ini to toml.
+      The new configuration documentation can be found at
+      <link xlink:href="https://naim94a.github.io/udpt/config.html">the official website</link> and example
+      configuration is packaged in <literal>${udpt}/usr/lib/udpt/udpt.toml</literal>.
```

Comment:
```suggestion
      configuration is packaged in <literal>${udpt}/lib/udpt/udpt.toml</literal>.
```

Reply:
I agree, is there a convention which files go where? Upstream uses `/usr/src` which didn't really convince me as well :)